src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -817,3 +817,7 @@ Prevent Long URL From Breaking Out of Container
         margin: 10px;
     }
 }
+
+.student_remind_tick {
+    width: 15px;
+}
```

Comment:
Will be moved to "Atomic classes" section as 'table-column-checkbox'.
What about centering, I'd like to use 'text-align: center', but it will be needed to override some bootstrap padding/margin for "checkbox" class.